### ISSUE TITLE
Doors, in item form

### ISSUE THREAD

**Issue by gettingusedto** [CleverRaven/Cataclysm-DDA#77258]
### Is your feature request related to a problem? Please describe.

I noticed that doors when deconstructed turns into their bare components, which i find odd. Surely it's not that hard to unscrew the hinges to uninstall the door and i bloody doubt folks woodwork their own doors every time they uninstall a door.

### Solution you would like.

Give doors an item form and make use of it in doors constructions.

### Describe alternatives you have considered.

Assume that the DDA-US people are just built different.

**Comment by PatrikLundell** [comment]
Yes, it makes sense. It's just that nobody has decided to spend the time to do the JSON work to implement it.

It may not be that much work to implement it (but there are a lot of different types of doors). You're certainly welcome to try to implement it.
